src/robot/libraries/Process.py

```diff
@@ -181,6 +181,20 @@ class Process(object):
     Note that the created output files are not automatically removed after
     the test run. The user is responsible to remove them if needed.
 
+    == Redirecting stdout or stderr using DEVNULL ==
+
+    In certain situations it is desirable to redirect ``stdout`` or ``stderr`` using a special value ``DEVNULL``.
+    ``DEVNULL`` can be used as the ``stdout`` or ``stderr`` argument and indicates that the
+    special file ``os.devnull`` will be used.  This is the equivalent of using ``> /dev/null`` on Unix-like machines or
+    ``> NUL`` on Windows machines.
+
+    Examples:
+    | ${result} = | `Run Process` | program | stdout=DEVNULL | stderr=${TEMPDIR}/stderr.txt |
+    | ${result} = | `Run Process` | program | stdout=${TEMPDIR}/stdout.txt | stderr=DEVNULL |
+    | ${result} = | `Run Process` | program | stdout=DEVNULL | stderr=DEVNULL |
+
+    Support for the special value ``DEVNULL`` is new in Robot Framework 3.2.
```

Comment:
This a very thorough documentation. I would actually even say it is too thorough as explaining anything about `os.devnull` is an implementation detail and `> /dev/null` can cause more confusing than bring benefits (especially when it only redirects stdout). I also feel that having a separate section about DEVNULL is also an overkill, similar STDOUT is just quickly mentioned in the general docs about stdout/stderr above.

My proposal is to add a new paragraph to the existing stdout/stderr section with a text like below (feel free to edit or rewrite totally):

> If you are not interested in outputs at all, you can explicitly ignore them by using a special value `DEVNULL` both with `stdout` and `stderr`. For example, `stdout=DEVNULL` is same as redirecting output on console with `> /dev/null` on Unix-like machines or `> NUL` on Windows.  This way the process will not hang even if there would be lot of output, but naturally output is not available after execution either. 

In addition to the explanation text, one example using DEVNULL should be added. There should also be a note about this being new in RF 3.2. It can either be a standalone note or the explanation text could have something like "Start from Robot Framework 3.2, ..." embedded to it.

Reply:
Ha ha, too much documentation - that's a first for me too!  I agree with your suggestion.  I will simply add a new paragraph, not a new section header and a single, simple example where it redirects both stdout and stderr using DEVNULL.